lib/livepush.ts

```diff
@@ -112,6 +112,16 @@ export class Livepush extends (EventEmitter as {
 		}
 	}
 
+	// Given some fs changes in the context, do we need to
+	// perform a livepush?
+	public livepushNeeded(addedOrUpdated: string[], deleted: string[]): boolean {
+		return !_.isEmpty(
+			this.dockerfile.getActionGroupsFromChangedFiles(
+				addedOrUpdated.concat(deleted),
```

Comment:
Would this style be cleaner, do you think?

```suggestion
				[...addedOrUpdated, ...deleted],
```

Reply:
I have no preference, can definitely change